helm/cluster-azure/values.schema.json

```diff
@@ -35,6 +35,30 @@
             "title": "Azure settings",
             "type": "object"
         },
+        "bastion": {
+            "properties": {
+                "enabled": {
+                    "title": "Enable bastion host for this cluster",
+                    "type": "boolean"
+                },
+                "instanceType": {
+                    "title": "Bastion Node instance size",
+                    "type": "string"
+                }
+            },
+            "title": "Bastion settings",
+            "type": "object"
+        },
+        "clusterDescription": {
+            "description": "User-friendly description of the cluster's purpose.",
+            "title": "Cluster description",
+            "type": "string"
+        },
+        "clusterName": {
+            "description": "Unique identifier, cannot be changed after creation.",
+            "title": "Cluster name",
+            "type": "string"
+        },
```

Comment:
Just noticed that this PR added clusterDescription and clusterName back, which have been moved to /metadata in a different PR.

Reply:
I'll make a PR to remove it.

Reply:
https://github.com/giantswarm/cluster-azure/pull/48

Reply:
oh sorry ... i thought i addressed it :+1: